components/brave_rewards/resources/ui/components/modalRedirect/style.ts

```diff
@@ -25,6 +25,7 @@ export const StyledLoader = styled('div')<{}>`
 
 export const StyledError = styled('div')<{}>`
   text-align: center;
+  white-space: pre-wrap;
```

Comment:
We're using this to preserve newlines from the message string? We might want `pre-line` if we are only interested in the newlines and not other whitespace.

Reply:
yes, just for the new lines. I'll update. Thanks!

Reply:
We already have a way of doing this (via `errorText?: string[]`), see `flaggedWalletModal`.

Reply:
Ah nice, thanks @szilardszaloki. Updated.